README.md

```diff
@@ -183,9 +184,19 @@ excluded:
 - App/HiDrive/Generated
 ```
 
-#### Optional: Call the SwiftLint script without using the setup script
+##### Optional: Call the SwiftLint script without using the setup script
 If you want to copy the SwiftLint configuration and lint the code without integrating the setup script you can call `Submodules/SMF-iOS-CommonProjectSetupFiles/SwiftLint/copy-and-run-swiftlint-config.sh` directly.
 
+### DrSwift
+
+[DrSwift](https://github.com/dduan/DrString) is integrated in SMF-iOS-CommonProjectSetupFiles itself. The current version is 0.4.2.
+
+#### Integrate it into the project
+
+DrString is configured to run when you execute `Submodules/SMF-iOS-CommonProjectSetupFiles/setup-common-project-files.sh` with [framework target configuration](#setup) parameters.
+
+**Make sure that `/.drstring.toml` is added to the gitignore file as the DrSwift configuration file be automatically copied from the repo into the projects base folder.**
```

Comment:
How could we do to not forget about this? How to remind developers about this?

Reply:
We can mention it in a weekly. I'm also very sure @Tomusm will notice when the FW PRs appear and `.drstring.toml` is checked it instead of an updated `.gitignore`.